mimic/model/clb_objects.py

```diff
@@ -287,6 +287,14 @@ def add_node(self, node_list, lb_id, current_timestamp):
             _verify_and_update_lb_state(self, lb_id, False, current_timestamp)
 
             if self.lbs[lb_id]["status"] != "ACTIVE":
+                # This is one possible error response per the "Rackspace Cloud
```

Comment:
This seems to be an error that only occurs with creating new load balancers (since otherwise, the load balancer doesn't need to assign any IPs). When adding nodes, if the load balancer is in ERROR state, it seems to be the "Load Balancer ... has a status of ERROR and is considered immutable." ( https://github.com/rackerlabs/mimic/issues/270.)

Could you clarify the goal of this PR?  Is it to inject arbitrary failure modes when adding a node to a CLB?